meta-mender-variscite/templates/local.conf.append

```diff
@@ -1,8 +1,19 @@
 
 # Appended fragment from meta-mender-community/meta-mender-variscite/templates
 
+# Setting systemd as default
+DISTRO_FEATURES:append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd" 
+
 DISTRO ?= 'fslc-x11'
 
+# Mender Server Variables
```

Comment:
that chunk can be removed, it should come from the generic Mender template

Reply:
Added in commit af7b582f8c497f08e7cff28f69acca8d73eb8b62